projects/generate_waveforms/pyproject.toml

```diff
@@ -13,6 +13,16 @@ python = ">=3.9,<3.11"
 [tool.poetry.dev-dependencies]
 pytest = "^6.2"
 
+[tool.typeo]
+outdir = "${OUT_DIR}"
+n_samples = 10000
+prior_file = "${PRIOR_FILE}"
```

Comment:
Is there a specific setting of priors that's worth checking in to the repo so that we can version it alongside everything else? These prior files are pretty small right?

Reply:
Yes that's a good idea. I'll put one together